log/test/.keep

```diff
@@ -0,0 +1 @@
+
```

Comment:
Do we need this file?

Reply:
this is for the tests to pass on github actions. otherwise there will be no folder for the log file to be created